Dockerfile

```diff
@@ -14,7 +14,8 @@ ENV NODE_PATH $NVM_DIR/versions/node/$NODE_VERSION/bin
 ENV PATH $NODE_PATH:$PATH
 RUN apt-get update -yq \
     && apt-get install curl gnupg python2 -yq \
-    && apt-get install python3 -yq
+    && apt-get install python3 -yq \
+    && apt-get install build-essential -yq
```

Comment:
Do you need `build-essential`?

Reply:
Yes. otherwise I need to run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before I can build.
And that runs in simulator mode....and that's slow(er)

Reply:
Is this something I should do in production?

Reply:
I don't think so. It runs fine on production.
So far I've only seen this go wrong on Apple M1 & M2 machines.